workspaces/ui-v2/src/lib/parse-diff.ts

```diff
@@ -97,35 +91,14 @@ export class ParsedDiff {
       currentSpecContext.currentSpecEndpoints
     );
 
-    invariant(
-      Boolean(location!.pathId),
```

Comment:
pathId is non-nullable via types. If it isn't we should update the typing instead